API/Model/CountryRepository.cs

```diff
@@ -36,9 +37,19 @@ public IEnumerable<Country> GetAllCountries()
 
         public Country? GetCountryOfTheDay()
         {
-            // TODO: Implement logic to get country of the day
+            
+            DateTime today = DateTime.Today;
+            string stringToBeHashed = today.Month.ToString() + today.Day.ToString();
+            int hashed = stringToBeHashed.GetHashCode();
+
+
+            List<Country> Countries = _dbContext.Countries.ToList();
+            int AmountOfCountries = Countries.Count();
+            string CountryCode = Countries[hashed % AmountOfCountries].Code;
+         
             return _dbContext.Countries
-                .Include (c => c.Data)
+                .Where(c => c.Code.Equals(CountryCode))
```

Comment:
Why is Equals() used here but == is used later?

Reply:
Because it is a string

Reply:
It is still different from what is used in the rest of the file, and the rest of the file works. Why not be consistent?

Reply:
Because i want it to be that way